tests/dast/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM golang:1.21
```

Comment:
I would like us to make use of their already existing image (it's the same dockerfile), and add on top of it what we need.

Reply:
At the moment we don't need anything , but i don't want to trust their docker file since they might add change things according to there testings.